native/chat/chat-thread-background.react.js

```diff
@@ -15,6 +20,35 @@ type BackgroundChatThreadListProps = {
 export default function BackgroundChatThreadList(
   props: BackgroundChatThreadListProps,
 ) {
+  const chatThreadListData = useSelector(state => chatListData(state));
+
+  const findUnreadBackgroundThreadsNumber = (
+    chatThreadsListData: Array<ChatThreadItem>,
+  ): number => {
+    return chatThreadsListData
+      .map(info => ({
+        home: info.threadInfo.currentUser.subscription.home,
+        unread: info.threadInfo.currentUser.unread,
+      }))
+      .filter(item => !item.home && item.unread).length;
+  };
+
+  const prevUnreadNumber = React.useRef(0);
+  const unreadBackgroundThreadsNumber = React.useMemo(() => {
+    return findUnreadBackgroundThreadsNumber(chatThreadListData);
+  }, [chatThreadListData]);
+
+  React.useEffect(() => {
+    if (unreadBackgroundThreadsNumber == prevUnreadNumber.current) return;
+    prevUnreadNumber.current = unreadBackgroundThreadsNumber;
+    let title = 'Background';
+    if (unreadBackgroundThreadsNumber !== 0)
+      title = title.concat(' (', unreadBackgroundThreadsNumber.toString(), ')');
+    props.navigation.setOptions({
+      title: title,
+    });
+  });
```

Comment:
Right now you've set this effect to run every time the component renders. Let's add a dep list here so it only updates when it needs to.

Reply:
Check if the dep lists in `useMemo` and `useEffect` are correct now, with selector used to get unread number